src/includes/replies/template.php

```diff
@@ -1364,6 +1363,11 @@ function bbp_get_reply_author_role( $args = array() ) {
 
 		$reply_id    = bbp_get_reply_id( $r['reply_id'] );
 		$role        = bbp_get_user_display_role( bbp_get_reply_author_id( $reply_id ) );
+		$css_role    = sanitize_key( $role );
+		
+		if( empty( $args['before'] ) ) {
```

Comment:
```suggestion
		if ( empty( $args['before'] ) ) {
```

Super tiny nitpick, but adding this space makes this line consistent with other similar lines.